components/Send/Send.tsx

```diff
@@ -88,15 +89,17 @@ const Send: React.FunctionComponent<SendProps> = ({
   const { decimalSeparator } = getNumberFormatSettings();
   // transparent is not spendable.
   const spendable = totalBalance.spendablePrivate + totalBalance.spendableOrchard;
-  const stillConfirming = parseFloat(spendable.toFixed(8)) !== totalBalance.total - totalBalance.transparentBal;
+
+  const stillConfirming =
+    parseFloat(spendable.toFixed(8)) !== totalBalance.total - (someUnconfirmed ? 0 : totalBalance.transparentBal);
```

Comment:
i think this would be more explicit if:
stillConfirming = spendablePrivate != privateBal || spendableOrchard != orchardBal || someUnconfirmed

Reply:
Thanks, A clean pair of eyes sometimes are really useful... you can see this more clearly than me... I will change this right away.